clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -302,6 +302,678 @@ RValue CIRGenFunction::buildRotate(const CallExpr *E, bool IsRotateRight) {
   return RValue::get(r);
 }
 
+/// Get builtin name for unhandled BuiltinID in the function
+/// `CIRGenFunction::buildBuiltinExpr`. Thoese builtin names are handled
+/// in OG code `CodeGenFunction::EmitBuiltinExpr`.
+/// NOTE: Every time we implement handling of a case in
+/// the function `CIRGenFunction::buildBuiltinExpr`, it should be remove here.
+static std::string getUnhandledBuiltinName(unsigned builtinID) {
```

Comment:
This should return StringRef.

Reply:
I'll use StringRef if we do return name.